fab/fabfile.py

```diff
@@ -948,6 +950,19 @@ def _migrate():
 
 
 @roles(ROLES_DB_ONLY)
+def _migrations_exist():
+    """
+    Check if there exists database migrations to run
+    """
+    _require_target()
+    with cd(env.code_root):
+        n_migrations = int(sudo(
+            '%(virtualenv_root)s/bin/python manage.py migrate --list | grep "\[ ]" | wc -l' % env)
+        )
+        return n_migrations > 0
```

Comment:
Probably not necessary to implement.  I was curious if there was a more precise way to check it, so I tracked this down.  Figured it was worth mentioning.